src/main/java/uk/gov/hmcts/reform/em/stitching/pdf/PDFMerger.java

```diff
@@ -108,14 +111,14 @@ private void addCoversheet(SortableBundleItem item) throws IOException {
 
             if (item.getSortedItems().count() > 0) {
                 pdfOutline.addParentItem(currentPageNumber, item.getTitle());
-            }
+                }
```

Comment:
Formatting issue here, set the correct indent

Reply:
done